apps/aecore/lib/aecore/structures/signed_tx.ex

```diff
@@ -4,9 +4,13 @@ defmodule Aecore.Structures.SignedTx do
   """
 
   alias Aecore.Keys.Worker, as: Keys
+  alias Aecore.Structures.TxData
   alias Aecore.Structures.SignedTx
 
-  @type signed_tx() :: %SignedTx{}
+  @type t :: %SignedTx{} | %SignedTx{
+    data: %TxData{},
+    signature: binary()
```

Comment:
The signature is supposed to be the "Signed %TxData{} with the private key of the sender". Whats the correct typespec for that?

Reply:
Ideally it should be

```
@type t :: %SignedTx{
    data: TxData.t(),
    signature: binary()
}
```

correct?

Reply:
The `SignedTx` represents the envelop for the `TxData` including the binary signature blob